internal/backend/db/dbgorm/ownership.go

```diff
@@ -10,6 +10,7 @@ import (
 
 	"go.autokitteh.dev/autokitteh/internal/backend/auth/authcontext"
 	"go.autokitteh.dev/autokitteh/internal/backend/db/dbgorm/scheme"
+	cctx "go.autokitteh.dev/autokitteh/internal/context"
```

Comment:
not sure cctx is a good convention
it's not really readable

you can go with akcontext or internalContext or something more descriptive and less error prone like ctx/cctx

Reply:
renamed